project-fob/Controllers/HomeController.cs

```diff
@@ -102,23 +92,14 @@ public ActionResult MeetingPageUser(string meetingId, string password)
                 if (password.Equals(meet.HostPassword.ToString()))
                 {
 
-                    User user = RetrieveUser();
-
-                    if (user == null)
-                    {
-                        user = new User(GenerateId());
-                        while (db.User.Any(m => m.UserId.Equals(user.UserId)))
-                        {
-                            user.UserId = GenerateId();
-                        }
-                        db.User.Add(user);
-                    }
-                    HttpContext.Session.Set("sessionid", Encoding.ASCII.GetBytes(user.UserId));
+                    CheckCookies();
+                    string id = GetCookieID();
```

Comment:
What's going on with this `CheckCookies()` and `GetCookieID()`? It seems a bit confusing to be using them like this.